lib/ukbus/include/uk/bus.h

```diff
@@ -42,9 +42,13 @@
 extern "C" {
 #endif
 
-#define UK_BUS_INIT_CLASS UK_INIT_CLASS_PLAT
-#define UK_BUS_INIT_PRIO  0
-#define UK_BUS_REGISTER_PRIO 0
+#define UK_BUS_INIT_CLASS UK_INIT_CLASS_SYS
```

Comment:
Do we know why it had PLAT prio before?

Reply:
Good point, I assumed that this was since the platform bus was introduced, but now you mention I had a closer look and see that the change was added in 3092e3ccd1a62406af2bd49d47496a0f9e2581de with some justification on why it cannot be `CLASS_EARLY`, but not much justification on why it's set to `CLASS_PLAT`. My understanding that devices are initialized at `CLASS_SYS` is because of the relevant comment in https://github.com/unikraft/unikraft/blob/staging/include/uk/init.h#L125:
>  Define a stage for device initialization

HOWEVER: I now notice that `CLASS_SYS` states https://github.com/unikraft/unikraft/blob/staging/include/uk/init.h#L104:

> Define a stage for platform initialization. Platform at this point read
> all the device and device are initialized.

What I propose is that I revert this change and move also the initialization of late UART from SYS to PLAT. And let's clarify the scope of these two init levels with @skuenzer when we re-arch ukboot.

Thanks for catching that!